### PR TITLE
Added styles for PRs page

### PR DESCRIPTION
**Pull request by Neelam618**

### CODE REVIEW

**Comment by virajvchavan** [comment]
Add one more feature:
- Highlight the top 3 users in the list

File: style.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
+
+ .hidden{
+    display: none;
+        }
```
Comment (virajvchavan) [inline style.css]: Fix indentation

File: style.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+body{
+	font-family: 'Ubuntu', sans-serif;
+	margin: 0;
+	padding: 0;
+	}
+#leaderboard-table{
+	width: 70%;
```
Comment (virajvchavan) [inline style.css:10]: Use class for the style. Don't use id for css

**Review by virajvchavan — CHANGES_REQUESTED** [review]

File: style.css (hunk 2 above)
Comment (Neelam618) [reply style.css:10]: done

**Pull request merged**